<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="PassGens">
      <attribute name="title">
       <string>Pass Gen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="general">
          <attribute name="label">
           <string>General</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Choose Password Generator to use</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cb_gens">
             <item>
              <property name="text">
               <string>PwGen</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pwqgen</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="le_customGen"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pwgen">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>508</width>
            <height>189</height>
           </rect>
          </property>
          <attribute name="label">
           <string>PwGen: Generate Pronounceable Passwords </string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Executable</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Pronounceable passwords are easier to memorize, Random are more secure</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="cb_randOrPron">
             <item>
              <property name="text">
               <string>Random</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pronounceable</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="le_pwgenpath">
             <property name="text">
              <string>/usr/bin/pwgen</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Length of password</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="le_pwgenLen">
             <property name="text">
              <string>16</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="le_pwgenCustom">
             <property name="text">
              <string>-s 16 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pb_pwgenpath">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="cbx_pwgenCustom">
             <property name="text">
              <string>Use custom options</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pcw">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>508</width>
            <height>189</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Page 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
